wheels/tests/model/sql/whereclause.cfc

```diff
@@ -53,5 +53,41 @@ component extends="wheels.tests.Test" {
 		actual = model("post").$whereClause(where="createdAtAlias > '#CreateDate(2000, 1, 1)#'");
 		assert("actual[4].datatype eq 'datetime'");
 	}
+	
+	function test_SQLInjectionProtectionWithParameterize() {
+		   badparams = {
+	    	username = "tonyp",
+	    	password = "tonyp123' OR password!='tonyp123"
+			};
+
+	    errorMessage = '';
+	    
+	    try {
+	    	inject = model("user").findall(where="username = '#badparams.username#' AND password = '#badparams.password#'", parameterize=2);
+	    } catch (any e) {
+	    	errorMessage = e.message;
+	    } 
+
+    	assert( errorMessage is "Wheels found 3 parameters in the query string but was instructed to parameterize 2.");
```

Comment:
This can also be done using the test framework's `raised()` function:
```
actual = raised("model('thisHasNoTable')");
expected = "Wheels.TableNotFound";
assert("actual eq expected");
```